shielddash/core/templates/shielddash/index.html

```diff
@@ -0,0 +1 @@
+{% extends 'shielddash/base.html' %}
```

Comment:
Do we plan to extend this in the future? If not it might be slightly more efficient to just load the template directly. Probably not that big of a deal but I'm a fan of not doing things like this until we actually need it.